src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -979,11 +995,13 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
                 std::cerr << inst->id() << std::endl;
             }
 
-            GPU_DEBUG_IF(debug_config->dump_layers_dst_only == 0 &&
-                            debug_config->is_dumped_layer(layer_name)) {
+            GPU_DEBUG_IF(debug_config->dump_layers_dst_only == 0 && debug_config->is_dumped_layer(layer_name)) {
                 for (size_t i = 0; i < get_primitive(inst->id())->dependencies().size(); i++) {
-                    log_memory_to_file(get_primitive(inst->id())->dep_memory_ptr(i), get_stream(),
-                                       layer_name + "_src_" + std::to_string(i));
+                    log_memory_to_file(get_primitive(inst->id())->dep_memory_ptr(i),
+                                       get_stream(),
+                                       "program_" + std::to_string(get_program()->get_id()) + "_" + layer_name +
```

Comment:
I think, to avoid conflict, it is necessary to use network id, too.. It is because a program may be instantiated into several networks.

Reply:
Done.